src/domain/collection/curatedEventList/utils.ts

```diff
@@ -0,0 +1,104 @@
+import * as React from 'react';
```

Comment:
perhaps the name of the file could be more descriptive. If it's only exporting a hook, it could be even the name of the function: `usePaginatedEventsByIdsQuery.ts` or something shorter

Reply:
Yes it is only exporting a hook atm but this is pretty much following the same pattern as other component folders, can't see any real benefit for a new file 🤔